### ISSUE TITLE
NFTs are not loaded 

### ISSUE THREAD

**Issue by selimerunkut** [selimerunkut/pixelk_website#34]
The nfts are not loaded, is this because of some throttling issue of the opensea API?
https://github.com/ProjectOpenSea/opensea-js/issues/86

or something else?

**Comment by deathlock** [comment]
Yes because of opensea's API restrictions or sometimes downtime.
